app/models/game.rb

```diff
@@ -3,6 +3,7 @@
 class Game < ApplicationRecord
   has_many :divisions, dependent: :destroy
   has_many :teams, through: :divisions, dependent: :destroy
+  has_many :users, through: :teams, dependent: :destroy
```

Comment:
While this is a true relation, I'm assuming this is being added now because of the bulk transcripts and resume download is added to the games controller. Why do we want to do all of this through a Game?